mantidimaging/gui/windows/live_viewer/model.py

```diff
@@ -114,8 +177,20 @@ def path(self, path: Path) -> None:
         self.image_watcher = ImageWatcher(path)
         self.image_watcher.image_changed.connect(self._handle_image_changed_in_list)
         self.image_watcher.recent_image_changed.connect(self.handle_image_modified)
+        self.image_watcher.update_spectrum.connect(self.presenter.update_spectrum)
         self.image_watcher._handle_notified_of_directry_change(str(path))
 
+    @property
+    def images(self):
+        return self._images if self._images is not None else None
+
+    @images.setter
+    def images(self, images):
+        self._images = images
+
+    def set_roi(self, roi: SensibleROI):
+        self.roi = roi
```

Comment:
I don't think these are needed. Could just let things directly access `images` and `roi` as attributes.